### PR TITLE
[docs]: Fixed the spellings in the docs

### PR DESCRIPTION
**Pull request by 2001asjad**

### CODE REVIEW

File: docs/docs/dashboard.md [hunk 1]
```diff
@@ -297,7 +297,7 @@ This option allows you to select a specific version of the app to export or expo
 
 #### Export ToolJet table schema
 
-Selecting this option will include the schema of the ToolJet table connected to that application in the exported JSON file. This option is available for all the apps on ToolJet however only the apps with a ToolJet table connected(includes tjdb query) will have the schema included in the exported JSON file.
+Selecting this option will include the schema of the tooljet table connected to that application in the exported JSON file. This option is available for all the apps on ToolJet however only the apps with a tooljet table connected(includes tjdb query) will have the schema included in the exported JSON file.
```
Comment (karan-rathod-316) [inline docs/docs/dashboard.md:300]: Please change tooljet to ToolJet

File: docs/docs/widgets/toggle-switch.md [hunk 2]
```diff
@@ -79,8 +79,8 @@ Under the <b>General</b> accordion, you can set the value in the string format.
 
 |  <div style={{ width:"100px"}}> Style </div> |  <div style={{ width:"100px"}}> Description </div> |  <div style={{ width:"100px"}}> Default Value </div>|
 |:----- |:---------  |:------------- |
-| Text color | Change the color of the text in the widget by providig the `Hex color code` or choosing a color from the picker. |  |
-| Toggle switch color | Change the color of the toggle switch in the widget by providig the `Hex color code` or choosing a color from the picker. |  |
+| Text color | Change the color of the text in the widget by providing the `Hex color code` or choosing a color from the picker. |  |
```
Comment (karan-rathod-316) [inline docs/docs/widgets/toggle-switch.md:82]: Please check if the formatting is correct here, I see double bars `| |` in the table.

File: docs/docs/widgets/multiselect.md [hunk 3]
```diff
@@ -40,7 +40,7 @@ For comprehensive information on all available **Actions**, refer to the [Action
 `await components.multiselect1.clearSelections()` <br/>
 `await components.multiselect1.deselectOption(2)`
 
-Following actions of multselect component can be controlled using the component specific actions(CSA):
+Following actions of multiselect component can be controlled using the component specific actions(CSA):
```
Comment (karan-rathod-316) [inline docs/docs/widgets/multiselect.md:43]: multiselect and all other components should always be in title case.

File: docs/docs/widgets/toggle-switch.md (hunk 2 above)
Comment (karan-rathod-316) [inline docs/docs/widgets/toggle-switch.md:82]: I see double bars here, is this correct?

`picker. |  |`

File: docs/docs/widgets/toggle-switch.md [hunk 4]
```diff
@@ -79,8 +79,8 @@ Under the <b>General</b> accordion, you can set the value in the string format.
...
+| Toggle switch color | Change the color of the toggle switch in the widget by providing the `Hex color code` or choosing a color from the picker. |  |
```
Comment (karan-rathod-316) [inline docs/docs/widgets/toggle-switch.md:83]: `from the picker. |  |`

Same issue as previously highlighted, please check if this is correct:

**Review by karan-rathod-316 — DISMISSED** [review]
Please go through the comments and ensure that changes are applied across docs.

File: docs/docs/widgets/toggle-switch.md (hunk 2 above)
Comment (2001asjad) [reply docs/docs/widgets/toggle-switch.md:82]: Yeah, so since we have created the table, the columns where there was no default value, were left blank.

**Review by amanregu — APPROVED** [review]

**Pull request merged**